clients/da-vinci-client/src/main/java/com/linkedin/davinci/kafka/consumer/StoreIngestionTask.java

```diff
@@ -395,6 +396,7 @@ public StoreIngestionTask(
     this.storeName = versionTopic.getStoreName();
     this.isUserSystemStore = VeniceSystemStoreUtils.isUserSystemStore(storeName);
     this.realTimeTopic = pubSubTopicRepository.getTopic(Utils.getRealTimeTopicName(version));
+    this.separateRealTimeTopic = pubSubTopicRepository.getTopic(Version.composeSeparateRealTimeTopic(storeName));
```

Comment:
Can we set this topic iff `isSeparatedRealtimeTopicEnabled` OR we can remove boolean for `isSeparatedRealtimeTopicEnabled` and use nullability of this field to determine if `isSeparatedRealtimeTopicEnabled`

Reply:
isSeparatedRealtimeTopicEnabled is used in multiple places, I can change this to be based on the flag. I created this variable here mainly for the fact that I don't want to create it everytime it is needed.

Reply:
We can keep the method: `isSeparatedRealtimeTopicEnabled` itself and change the impl to use nullness check? and get rid of var `isSeparatedRealtimeTopicEnabled`